lib/zone-spec/long-stack-trace.ts

```diff
@@ -136,6 +136,151 @@ function captureStackTraces(stackTraces: string[][], count: number): void {
   }
 }
 
+const zoneAwareStackFrames = {};
```

Comment:
Yes, sure, I put it into longStackTrace because it can catch the stack frames for onScheduleTask, and I realize I can just create a error in onScheduleTask callback, I will move those code into zone.ts